DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -545,21 +550,26 @@ static function bool BingoGoalImpossible(string bingo_event, int start_map, int
         case "UNATCOHandbook":
         case "ManderleyMail":
         case "LetMeIn":
+        case "ViewPortraits": // next location is 04_NYC_Bar
             return start_map > 30 && start_map < 36 && end_mission <= 3;// you can do these m03 unatco goals in m04 unatco, but if you start in helibase it's far
```

Comment:
30 is UNATCO Island, and 31 is UNATCO HQ, right?

```suggestion
            return start_map > 31 && start_map < 36 && end_mission <= 3;// you can do these m03 unatco goals in m04 unatco, but if you start in helibase it's far
```

Reply:
Indeed. I neglected to update that check with the one before it.